sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -3098,4 +3098,20 @@ class HiveDDLSuite
       "CREATE TABLE tab (c1 int) PARTITIONED BY (c1) STORED AS PARQUET",
       "Cannot use all columns for partition columns")
   }
+
+  test("SPARK-43359: Delete table not allowed") {
+    val tbl = "T1"
+    withTable(tbl) {
```

Comment:
Where is `tbl` used besides of `withTable `? Could you use it in the sql statements below, please.

Reply:
I have already fixed it